### PR TITLE
[SELS-TASK] Create Quiz (admin) list markup (table, record-actions)

### PR DESCRIPTION
**Pull request by jstephend-sun**
Asana [Task](https://app.asana.com/0/1201478050789792/1201478462951337/f)

commands:
```
npm i
npm start
```

expected output: markup only with some sample data

screenshots:
image

### CODE REVIEW

File: frontend/src/components/Table/index.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  {
+    name: 'Jane Cooper',
+    title: 'Regional Paradigm Technician',
+    department: 'Optimization',
+    role: 'Admin',
+    email: 'jane.cooper@example.com',
+    image:
+      'https://images.unsplash.com/photo-1494790108377-be9c29b29330?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=facearea&facepad=4&w=256&h=256&q=60',
+  },
+];
```
Comment (kimjo77) [inline frontend/src/components/Table/index.tsx:21]: do you still need this file? i could not find where you called and used this file.

Comment (jstephend-sun) [reply frontend/src/components/Table/index.tsx:21]: yes po, the use is present on later PRS,

one of the PR include the refactor for the imports

image
image

Comment (jstephend-sun) [reply frontend/src/components/Table/index.tsx:21]: PR 15 was the pr where i mentioned i refactored the imports

Comment (kimjo77) [reply frontend/src/components/Table/index.tsx:21]: ahh okay then this is resolved :bow:

**Review by kimjo77 — APPROVED** [review]

**Pull request merged**